Description: avoid worldlist*zip and hunspell*zip for hunspell
 Let's not create worldlist*zip and hunspell*zip for hunspell. We are not going
 to distribute them or use them in any way. Saves us a zip build-depdency
 but more importantly it does not create "binary" files in speller/hunspell
 we would need to manually remove it clean.
Author: Rene Engelhard <rene@debian.org>

---
Origin: vendor
Forwarded: no
Last-Update: 2017-10-20

--- a/speller/make-hunspell-dict
+++ b/speller/make-hunspell-dict
@@ -56,12 +56,8 @@
     cat $3 >> README_$1.txt
   fi
 
-  rm -f hunspell-$fn.zip
-  zip -9 hunspell-$fn.zip README_$1.txt $1.dic $1.aff
-
   if [ -z "$3" ]; then
     mkdir -p hunspell
-    cp hunspell-$fn.zip hunspell/
 
     #echo check
 
@@ -70,11 +66,6 @@
     #hunspell -l -d ./$1 < $1.dic.tocheck > misspelled
 
     cat $1.tocheck | iconv -f iso-8859-1 -t utf-8 | $UNIX2DOS > $1.txt
-    cat <<EOF > README.txt
-This zip file contains the words found in the corresponding Hunspell
-dictionary.  See the file README_$1.txt.
-EOF
-    zip -9 hunspell/wordlist-$fn.zip README.txt README_$1.txt $1.txt
   fi
 }
 
